components/SearchBar.vue

```diff
@@ -0,0 +1,76 @@
+<template>
+    <div class="flex flex-row flex-0 w-128">
+        <div id="search-fields" class="grid-cols-3 mx-4">
+            <div class="search-specialty col-span-1 inline-block w-1/3 py-4">
+                <select class="rounded-l-full rounded-r-none w-full px-1
+                    border-2 border-primary/80 py-1.5 drop-shadow-md text-primary-text bg-secondary-bg/5">
+                    <option value="" class="text-primary-text-muted" disabled selected>
+                        Select a specialty...
+                    </option>
+                    <option value=""></option>
+                    <option :key="index" v-for="(specialty, index) in specialtyDropdownOptions">
+                        {{ specialty.names[0].name }}
+                    </option>
+                </select>
+            </div>
+            <div class="search-location col-span-1 inline-block w-1/3 py-4">
+                <select class="rounded-none w-full px-1
+                    border-2 border-primary/80 py-1.5 drop-shadow-md text-primary-text bg-secondary-bg/5">
```

Comment:
```suggestion
                    border-y-2 border-primary/80 py-1.5 drop-shadow-md text-primary-text bg-secondary-bg/5">
```

Reply:
I would leave out the border on the left and right sides to prevent the lines from doubling from the surrounding two fields.

Reply:
Fair enough 👌